src/EventStore.Projections.Core.Tests/Services/v8/when_running_a_projection_emitting_malformed_events.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
```

Comment:
I can't tell the difference between these two tests

Reply:
Technically, there are the same as `linkTo` uses `emit` internally. The `emit` version of the test checks if the event itself is valid whereas the `linkTo` version goes deeper. While the event itself is correct, if we look at it as a link event, that event is no longer valid. By having those two tests, I make sure those two invariants remain checked at all time at the cost of a small overlap.